cypress/integration/inventory/create-page-type-request.spec.js

```diff
@@ -83,7 +83,7 @@ describe('ui-inventory: Create page type request', () => {
     Requests.deleteRequestPolicyApi(requestPolicyId);
   });
 
-  it('C546: Create new request for "Page" type (folijet) (vega)', { tags: [TestTypes.smoke, DevTeams.folijet] }, () => {
+  it('C546: Create new request for "Page" type (folijet) (vega)', { tags: [TestTypes.smoke, DevTeams.folijet, DevTeams.vega] }, () => {
```

Comment:
Does it Vega?

Reply:
please don't add the tag for Vega it will be problem for me to find this test and to fix

Reply:
Done